rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -47,56 +61,43 @@ class TrainerRunner:
 
     def __init__(
         self,
-        trainer_class: Type[RLTrainer],
-        trainer_config: Mapping[str, Any],
-        compute_config: Mapping[str, Any],
+        rl_trainer_spec: RLTrainerSpec,
     ):
-        num_gpus = compute_config.get("num_gpus", 0)
-        use_fake_gpus = compute_config.get("_use_fake_gpus", False)
-        self._trainer_config = trainer_config
-
-        if num_gpus > 0:
-            scaling_config = ScalingConfig(
-                num_workers=num_gpus,
-                use_gpu=(not use_fake_gpus),
-            )
-
-            if trainer_class.framework == "torch":
-                from ray.train.torch import TorchConfig
-
-                backend_config = TorchConfig()
-            elif trainer_class.framework == "tf":
-                from ray.train.tensorflow import TensorflowConfig
+        scaling_config = rl_trainer_spec.trainer_scaling_config
+        rl_trainer_class = rl_trainer_spec.rl_trainer_class
 
-                backend_config = TensorflowConfig()
-            else:
-                raise ValueError("framework must be either torch or tf")
+        # TODO (Kourosh): Go with a _remote flag instead of _is_local to be more
```

Comment:
get rid of this TODO or just go with it?

Reply:
Let's leave it as todo, cuz I like your suggestion a bit better. _remote is more clear mode vs. _local mode. local name is a bit generic. But remote is a ray term.